rmgpy/rmg/rmgTest.py

```diff
@@ -42,6 +42,7 @@
 from rmgpy.data.base import ForbiddenStructures
 
 from rmg import *
```

Comment:
Do we still need 'from rmg import *?` After changing, `main()` is the only function imported and is not used in `rmgTest.py`